### PR TITLE
Create 2019-04-03-co-operatives-workshop-in-oakland.md

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

**Comment by dcwalk** [comment]
I'm not really sure what this is @benhylau -- do we need the notes here?

**Comment by benhylau** [comment]
I feel it's relevant same as the root systems call @dcwalk 
We could restructure this repo / rearrange things

File: 2019-04-03-co-operatives-workshop-in-oakland.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
+
+Time: 18:30 PST  
+Location: Oakland  
+Attending: Danny Spitzberg, Eugene, Pam, Ben + 3 other workshop participants
+
+[TOC]
+
+## Notes
+
```
Comment (benhylau) [inline 2019-04-03-co-operatives-workshop-in-oakland.md]:
```suggestion

- Ben learnt of a co-operative workshop through Pam, hosted by Danny and Eugene, so they went...
```

**Comment by benhylau** [comment]
Good point, I added something. Still think we should put all these "research" into a folder

**Comment by dcwalk** [comment]
@benhylau looks good! I think you are right we need to do a regig of this folder, but I don't have a sense of best flow! If you do, why don't we merge this and then do a clean up seperately?

**Comment by patcon** [comment]
fwiw I met Danny in Taipei during a post-summit meeting, the point of which I understood was to build solidarity between north american devs and taiwanese. cooperatives came up a few times. Notes here: https://hackmd.io/LTyxOzBjS7WAoefBBrVPGA

**Comment by patcon** [comment]
Oh hey, was this actually on Apr 4th? If so, we should correct to 2019-03-04, right?

**Pull request merged**